charts/retool/templates/deployment_workflows_worker.yaml

```diff
@@ -336,9 +340,13 @@ spec:
   - protocol: TCP
     port: 3005
     targetPort: 3005
+    {{ if .Values.workflows.worker.portName }}
+    name: {{ .Values.workflows.worker.portName }}
+    {{ else }}
     name: {{ template "retool.name" . }}
```

Comment:
Maybe you can do the `| default` here too

https://helm.sh/docs/chart_template_guide/functions_and_pipelines/#using-the-default-function

Reply:
line 346 is the fallback if `.Values.workflows.worker.portName` is not set, so we should be good here?

Reply:
I mean like you do above - the conditional L343-L347 becomes one line

`name: {{ .Values.workflows.worker.portName | default (template "retool.name" .) }}`